src/lm_robust_helper.cpp

```diff
@@ -243,16 +243,16 @@ List lm_solver(Eigen::Map<Eigen::MatrixXd>& Xfull,
 
        // iterate over unique cluster values
        for(int i = 1;
-           i < n;
+           i <= n;
```

Comment:
for loops can be one line.

Reply:
lol no idea why it's more than one line. Must have had something long previously....